src/appleseed/renderer/modeling/material/material.cpp

```diff
@@ -94,6 +94,20 @@ bool Material::has_alpha_map() const
     return get_non_empty(m_params, "alpha_map") != 0;
 }
 
+bool Material::has_uniform_opacity() const
```

Comment:
I think we can rename this method. It's actually returning true if the material has an uniform alpha map with value 1.
I'd also not use the word opacity.